develop/integrations/.pages

```diff
@@ -3,4 +3,4 @@ nav:
   - index.md
   - 'Wallets': wallets.md
   - 'Indexers': indexers.md
-  - 'Others': others.md
\ No newline at end of file
+  - 'Oracles': oracles.md
```

Comment:
This is low priority, but would it make sense to order these pages alphabetically? As they don't need some sort of storytelling, maybe we can use that criteria to order them. WDYT?

Reply:
Agree with this